src/main/java/au/org/emii/geoserver/extensions/filters/layer/data/ValueEncoder.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2014 IMOS
+ *
+ * The AODN/IMOS Portal is distributed under the terms of the GNU General Public License
+ *
+ */
+
+package au.org.emii.geoserver.extensions.filters.layer.data;
+
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.Set;
+import java.util.TimeZone;
+
+public class ValueEncoder {
+
+    List<String> encodeValues(Set values) {
+        List<String> result = new ArrayList<String>();
+
+        for (Object value : values) {
+            result.add(encodeValue(value));
+        }
+
+        return result;
+    }
+
+    String encodeValue(Object unencoded) {
```

Comment:
Also, if we were going to rename this class we would also want to rename this method.